### ISSUE TITLE
class was missing while executing org.jooq:jooq-codegen-maven

### ISSUE THREAD

**Issue by baiyapeng** [jOOQ/jOOQ#7848]
```
ERROR] Failed to execute goal org.jooq:jooq-codegen-maven:3.10.8:generate (default-cli) 
                on project jooq: Execution default-cli of goal org.jooq:jooq-codegen-maven:3.10.8:generate 
                failed: A required class was missing while executing org.jooq:jooq-codegen- 
                 maven:3.10.8:generate: javax/xml/bind/JAXB
[ERROR] -----------------------------------------------------
[ERROR] realm =    plugin>org.jooq:jooq-codegen-maven:3.10.8
.....
```

but，I am sure that JAXB is in the classpath.
I am sad，anyone can help me，very thanks

- jOOQ:3.10.8
- Java:8
- Database:mysql-5.7-docker-edtion
- OS:mac
- JDBC Driver:mysql-connector-java-6.0.6

**Comment by lukaseder** [comment]
I cannot reproduce it as it stands now. Feel free to reopen with details, if you think this is still a problem.

**Comment by cg-d** [comment]
Ok, when switching to 3.11.4 the plugin is working. The version shipped with Spring Boot 2.0.6 is not.
